src/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -64,7 +66,13 @@ window.__KBN__ = {
   }
 };
 
-require('ui/test_harness');
-${bundle.getRequires().join('\n')}
-require('ui/test_harness').bootstrap(/* go! */);
+new CoreSystem({
+  injectedMetadataForTesting: {
+    legacyMetadata
+  }
+}).initLegacyPlatform(() => {
```

Comment:
The convention we've talked about is having a core `start` function and then invoking legacy startup/bootstrap through a legacy service within the core system. This way we can identify precisely when the legacy code kicks off in the context of all the new platform services we'd be rolling out.

Is there a reason we need to abandon that approach for this initLegacyPlatform approach?

Reply:
Nope, happy to move it there.

Reply:
I just didn't want to worry about start()/stop() right now, but it's not a big deal.